src/main/resources/kafka-topics.json

```diff
@@ -0,0 +1,9 @@
+{
+  "topics": [
+    {
+      "name": "inventory.instance",
```

Comment:
This name is specific to the domain, however it does not mention storage. If mod-inventory were to publish higher level, domain meaningful events (rather than the change messages here), would we expect them to be published to the same topic?